lib/client/api.go

```diff
@@ -391,57 +391,66 @@ func MakeDefaultConfig() *Config {
 // SSH certificate.
 type ProfileStatus struct {
 	// Name is the profile name.
-	Name string
+	Name string `json:"name"`
```

Comment:
I am not sure TBH that we want to marshal this struct directly.

This is a part of the internal API and as soon as we start marshaling it and showing to users, it becomes a part of the public API so we'd have to keep this in mind and not make backwards-incompatible changes which isn't always possible. lib/client is supposed to be internal. Can we instead define a user-facing profile struct that tsh would marshal for the user, maybe with some irrelevant fields omitted?

Think of it as lib/client being the backend and tsh being the frontend. You usually don't return backend objects directly to the client.